File: RedSheet.DbEntities/Models/vSayingOpinion.cs

using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations.Schema;
using System.ComponentModel.DataAnnotations;
using Microsoft.EntityFrameworkCore;
using Rx.Core.Data.Attributes;
using RedSheet.DbEntities.Enums;

namespace RedSheet.DbEntities.Models
{
	[Table("vSayingOpinions", Schema = "dbo")]
	[Serializable]
	public partial class vSayingOpinion  
	{
 
	[DatabaseGenerated(DatabaseGeneratedOption.None)] 
	[Key]
	public int CultureCountryId { get; set; } 
 
	[Range(1, int.MaxValue)]
	public int SayingOpinionId { get; set; } 
 
	[Required] 
	[MaxLength(500)]
	public string SayingOpinionValue { get; set; } 

	public vSayingOpinion()
	{
    }

    }
}

File: redsheet-portal/src/app/database-models/v-saying-opinion.ts
import { required, maxLength, range } from '@rx/annotations';

export class vSayingOpinion {
    constructor(vSayingOpinion?: vSayingOpinion )  {
        let properties = [ "cultureCountryId", "sayingOpinionId", "sayingOpinionValue",];
        for (let property of properties)
            if (vSayingOpinion && vSayingOpinion[property])
                this[property] = vSayingOpinion[property];
    }
 
	cultureCountryId : number =   0 ;
 
    @range(1,2147483647)
	sayingOpinionId : number =   undefined;
 
    @required()
    @maxLength(500)
	sayingOpinionValue : string =   undefined;
}
